### ISSUE TITLE
qs object not shown in post - Uncaught TypeError: qlik.setOnError is not a function

### ISSUE THREAD

**Issue by nixnut** [yianni-ververis/qlik-sense-wordpress-plugin#1]
When adding a qs object in a blog post the object is not shown when viewing the post. In the console the following error is shown:

```
index.js:13 Uncaught TypeError: qlik.setOnError is not a function
    at index.js:13
    at Object.execCb (require.js:6)
    at t.check (require.js:6)
    at t.<anonymous> (require.js:6)
    at require.js:6
    at require.js:6
    at each (require.js:6)
    at t.emit (require.js:6)
    at t.check (require.js:6)
    at t.enable (require.js:6)
    at t.init (require.js:6)
    at o (require.js:6)
    at Object.completeLoad (require.js:6)
    at HTMLScriptElement.onScriptLoad (require.js:6)
```

However on the homepage the qs object does get rendered. But in the console another error is shown.

See the attached screenshots and logs below.

[wp-blogpost-console.log](https://github.com/yianni-ververis/qlik-sense-wordpress-plugin/files/1464466/wp-blogpost-console.log)
[wp-homepage-console.log](https://github.com/yianni-ververis/qlik-sense-wordpress-plugin/files/1464467/wp-homepage-console.log)
![2017-11-12 09_45_09-hello world worpress](https://user-images.githubusercontent.com/17782405/32697431-8cb50112-c790-11e7-9c63-940e51c4880a.png)
![2017-11-12 09_46_45-wptest_gdw wp homepage](https://user-images.githubusercontent.com/17782405/32697434-926afec2-c790-11e7-8663-56c90168c553.png)

**Comment by yianni-ververis** [comment]
For the first is definitely qlik.js is not loaded. What is your virtual proxy in your settings?

**Comment by nixnut** [comment]
Exactly the same as in the second since it's a site-wide setting. It's /anon/
I can't image the plugin settings are incorrect since the qs object is shown in the post on the wordpress homepage. Just on the blog post page itself things don't go as expected.

**Comment by yianni-ververis** [comment]
Did you try to upgrade to 1.1.1?

**Comment by nixnut** [comment]
Yes, I have. The error _index.js:13 Uncaught TypeError_ is still there and the qs object is not rendered on the post page.

**Comment by nixnut** [comment]
I've narrowed down the problem a bit. It appears that QS objects are not shown on the blog post page when I'm logged in and can edit the post. When I'm not logged in the QS objects are shown on the blog post page as well as on the site homepage. I have very little knowledge of Wordpress, but I suspect the loading of the js resources is slightly different which results in the error. Perhaps it's possible to defer the loading and rendering of the QS objects until wordpress has done its own stuff.

**Comment by yianni-ververis** [comment]
Thank you for your contribution. I assume that its fixed with v1.1.5 so I close this issue
